src/util/system.cpp

```diff
@@ -973,8 +980,8 @@ bool ArgsManager::ReadConfigFiles(std::string& error, bool ignore_invalid_keys)
         m_config_sections.clear();
     }
 
-    const fs::path conf_path = GetPathArg("-conf", BITCOIN_CONF_FILENAME);
-    std::ifstream stream{GetConfigFile(conf_path)};
+    auto conf_path{GetConfigFilePath()};
```

Comment:
sorry, forgot to mark this as `const` in my diff
```suggestion
    const auto conf_path{GetConfigFilePath()};
```

Reply:
sure, also agree